app/web_conn.go

```diff
@@ -749,6 +749,13 @@ func (wc *WebConn) shouldSendEvent(msg *model.WebSocketEvent) bool {
 		return wc.GetConnectionID() == msg.GetBroadcast().ConnectionId
 	}
 
+	// if the connection is omitted don't send the message
+	if msg.GetBroadcast().OmitConnectionId != "" {
+		if wc.GetConnectionID() == msg.GetBroadcast().OmitConnectionId {
+			return false
+		}
+	}
```

Comment:
Small detail but doing an `&&` will save us some unnecessary indentation. Plus if we can assume `wc.GetConnectionID()` will never return empty then it's just one equality check.